code/game/turfs/simulated/minerals.dm

```diff
@@ -695,9 +696,12 @@
 	. = ..()
 
 /turf/closed/mineral/gibtonite/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/t_scanner/adv_mining_scanner/goat_scanner))
```

Comment:
needs a stage check like the next if statement, looks like you can defuse one that isn't going off...